include/json/value.h

```diff
@@ -171,6 +171,9 @@ class JSON_API Value {
     CZString(ArrayIndex index);
     CZString(const char* cstr, DuplicationPolicy allocate);
     CZString(const CZString& other);
+#ifdef JSON_HAS_RVALUE_REFERENCES
+	CZString(CZString&& other);
```

Comment:
A `#if` shouldn't affect indentation of C++ declarations. This should be aligned with the other constructors.
Should be marked `noexcept`.